Formula/m/maturin.rb

```diff
@@ -16,20 +16,28 @@ class Maturin < Formula
     sha256 cellar: :any_skip_relocation, x86_64_linux:   "75672cf4f41c9fa3814a499065b6aed41a2a7001fbd9a194586063c229f25bcd"
   end
 
+  depends_on "python@3.9" => :test
+  depends_on "python@3.10" => :test
   depends_on "python@3.11" => :test
   depends_on "rust"
 
+  def pythons
+    deps.map(&:to_formula).sort_by(&:version).filter { |f| f.name.start_with?("python@") }
+  end
+
   def install
     system "cargo", "install", *std_cargo_args
```

Comment:
this isn't currently installing any python bindings, just the `maturin` binary

we need to `pip install` for each python